### PR TITLE
update UI for manage builds modal and labels

### PR DESCRIPTION
**Pull request by christianvogt**
## Fixes 

Manage builds model and label updates as per

## Description

Updated all pipeline build status labels with new content and tooltips.
Added `Roll back to default pipeline` kebab menu item to manage build pipelines.
Update microcopy for modal.

## Type of change

- [x] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Note: Updated close button to secondary variant but did not produce new screenshots.

image
image
image
image

image

image

image
image
image

removed `Merge build PR` filter:
image

## How to test or reproduce?
Put component into various states of upgrading to custom build pipelines.

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by christianvogt** [comment]
fyi @Ranelim

**Comment by christianvogt** [comment]
rebased PR

File: src/components/Components/ComponentPACStateLabel.tsx [hunk 1]
```diff
@@ -19,87 +20,73 @@ const ComponentPACStateLabel: React.FC<Props> = ({ component, onStateChange }) =
...
+    [enableAction, showModal, component.metadata.name, component.metadata.namespace],
   );
 
+  const style = customizePipeline ? { cursor: 'pointer' } : undefined;
   switch (pacState) {
     case PACState.sample:
       return (
-        <Tooltip content="You cannot customize the build pipeline of a sample. Fork the sample to your own repository and create a new component.">
-          <Label>Sample</Label>
+        <Tooltip content="This component currently uses our sample build pipeline. To enable automatic rebuild on commits to your main branch and pull requests, fork this sample and upgrade to a custom build pipeline by forking this repository.">
```
Comment (karthikjeeyar) [inline src/components/Components/ComponentPACStateLabel.tsx:41]:
```suggestion
        <Tooltip content="This component currently uses our sample build pipeline. To enable automatic rebuild on commits to your main branch and pull requests, fork this sample and upgrade to a custom build pipeline.">
```

File: src/components/CustomizedPipeline/CustomizePipelines.tsx [hunk 2]
```diff
@@ -26,18 +28,51 @@ import { ComponentKind } from '../../types';
...
+      onSelect={() => setOpen(false)}
+      toggle={<KebabToggle onToggle={() => setOpen((v) => !v)} id="toggle-id-6" />}
+      isOpen={isOpen}
+      isPlain
+      position={DropdownPosition.right}
+      dropdownItems={[
+        <DropdownItem
+          key="roll-back"
+          isDisabled={![PACState.error, PACState.pending, PACState.ready].includes(state)}
+          onClick={() => disablePAC(component)}
```
Comment (rottencandy) [inline src/components/CustomizedPipeline/CustomizePipelines.tsx:54]: Do we need access check for this?

Comment (christianvogt) [reply src/components/CustomizedPipeline/CustomizePipelines.tsx:54]: Ah yes. Your changes went in. I'll update this.

File: src/components/Components/ComponentPACStateLabel.tsx (hunk 1 above)
Comment (christianvogt) [reply src/components/Components/ComponentPACStateLabel.tsx:41]: Accepted suggestion.

**Pull request merged**